landoapi/api/landings.py

```diff
@@ -71,3 +69,25 @@ def get(landing_id):
         )
 
     return landing.serialize(), 200
+
+
+def update(landing_id, data):
+    """Update landing on pingback from Transplant."""
+    try:
+        landing = Landing.query.filter_by(
+            id=landing_id, request_id=data['request_id']
+        ).one()
+    except NoResultFound:
+        return problem(
+            404,
+            'Landing not found',
+            'The requested Landing does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+
+    landing.error = data.get('error_msg', '')
+    landing.result = data.get('result', '')
+    landing.status = TRANSPLANT_JOB_LANDED if data['landed'
```

Comment:
This line confuses me.  If the 'landed' field is present, regardless of value, everything is OK, and if an update is ever made with this field missing, it's a failure in the other system?  I think there should be a comment or two in here to explain what the possible combinations of incoming values are, and to explain what they represent in the other system and in our system.